net-libs/tox/tox-0.2.9.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils systemd user
+
+MY_P="c-toxcore-${PV}"
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI="https://github.com/TokTok/c-toxcore/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0/0.2"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+av daemon ipv6 log-debug +log-error log-info log-trace log-warn static-libs test"
+
+REQUIRED_USE="( ?? ( log-debug log-error log-info log-trace log-warn ) )"
+
+RDEPEND="
+	av? ( media-libs/libvpx:=
+		media-libs/opus )
+	daemon? ( dev-libs/libconfig )
+	>=dev-libs/libsodium-0.6.1:=[asm,urandom,-minimal]"
+BDEPEND="${RDEPEND}
```

Comment:
`BDEPEND` never inherits `RDEPEND`, it does not make sense. https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html

You need to keep `DEPEND` in this line instead, but move out `pkgconfig` to `BDEPEND`.